src/main/java/org/jboss/xavier/analytics/pojo/output/WorkloadInventoryReportModel.java

```diff
@@ -0,0 +1,215 @@
+package org.jboss.xavier.analytics.pojo.output;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.transaction.Transactional;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@Entity
+@Transactional
+public class WorkloadInventoryReportModel
+{
+    static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = javax.persistence.GenerationType.AUTO, generator = "WORLOADINVENTORYREPORTMODEL_ID_GENERATOR")
+    @GenericGenerator(
+            name = "WORLOADINVENTORYREPORTMODEL_ID_GENERATOR",
+            strategy = "org.hibernate.id.enhanced.SequenceStyleGenerator",
+            parameters = {
+                    @Parameter(name = "sequence_name", value = "WORLOADINVENTORYREPORT_SEQUENCE")
+            }
+    )
+    private Long id;
+
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "analysis_id")
+    @JsonBackReference
+    private AnalysisModel analysis;
+
+    private String provider;
+    private String datacenter;
+    private String cluster;
+    private String vmName;
+    private String osName;
+    private String osDescription;
+    private BigDecimal diskSpace;
+    private Integer memory;
+    private Integer cpuCores;
+    @ElementCollection(fetch = FetchType.EAGER)
+    private Set<String> workloads;
+    private String complexity;
```

Comment:
@mrizzi Do you think it is worth changing the type of `complexity` field to an `Enum` instead of `String`? I think the possible values of `complexity` is limited.

After this question is answered I think I will merge your PR. I think that travis CI is failing because of external problems

Reply:
Answer: Once we will be more stable we should make the complexity to become an enum but for now let's keep it as it is defined.

Reply:
Thanks @carlosthe19916 for reporting this and i can confirm once we will be sure about the `complexity` values and meaning, we will move to have it better defined using an enumeration.